repo/packages/K/kubernetes/1/config.json

```diff
@@ -81,6 +81,15 @@
           "minimum": 0,
           "type": "integer"
         },
+        "resource-roles": {
+          "default": ["slave_public"],
```

Comment:
I don't like the default value of "slave_public". public is something that people should opt-in to. default should be `*` and our docs (description) can suggest the `slave_public` alternative.